integration-tests/contract-transfer/lib.rs

```diff
@@ -252,7 +254,7 @@ pub mod give_me {
             let transfer = call.give_me(120);
 
             let call_res = client
-                .call(&ink_e2e::eve(), &transfer, 0, None)
+                .call(&ink_e2e::eve(), &transfer, 0, None, None)
```

Comment:
As far as I can see we are not using this API anywhere in the examples.

If we are going to move to this API and have it a breaking change anyway, we can just change `call` to return the call builder, so the default becomes in this case:

```
let call_res = client
  .call(&ink_e2e::eve(), &transfer)
  .submit()
```